### PR TITLE
prepare release v1.12.0

### PR DESCRIPTION
**Pull request by ammolitor**
prepare release v1.12.0

### CODE REVIEW

File: docs/changelog/v1.12.0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+### Operator
+
+* Remove old Kubernetes version from tests [#1471](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1471)
+* Add buggify option to ingore processes during restart [#1467](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1467)
+* Add docs for DNS setup  documentation Improvements or additions to documentation [#1466](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1466)
+* Small clean up [#1463](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1463)
+* Add first status check for the analyze command [#1461](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1461)
+* Security Updates [#1457](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1457)
+* Add update hooks to the mock Kubernetes client [#1456](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1456)
+* Ensure that Pods that are stuck in terminating but still have a fdbserver process reporting are marked as failed  [#1455](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1455)
```
Comment (johscheuer) [inline docs/changelog/v1.12.0.md:14]:
```suggestion
* Ensure that Pods that are stuck in terminating but still have a fdbserver process reporting are marked as failed [#1455](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1455)
```

**Review by johscheuer — APPROVED** [review]
LGTM 👍 One small style nit.

**Review by johscheuer — APPROVED** [review]
We can ignore the `Lint and Test Charts / lint-test (push)` failure. I believe this job never really worked.

**Comment by JoshuaMcManus** [comment]
> We can ignore the `Lint and Test Charts / lint-test (push)` failure. I believe this job never really worked.

@johscheuer can we get an issue filed to track the work to clean these failed test suites?

**Comment by johscheuer** [comment]
> > We can ignore the `Lint and Test Charts / lint-test (push)` failure. I believe this job never really worked.
> 
> @johscheuer can we get an issue filed to track the work to clean these failed test suites?

Yes, I file an issue for that one.

**Pull request merged**